client/app/api/course/Assessment/Submission/Answer/TextResponse.js

```diff
@@ -0,0 +1,72 @@
+import BaseAssessmentAPI from '../../Base';
```

Comment:
If this is a new file, why write it in JS?

Reply:
so sorry, I was prioritising the workability when I devised this (have problem in deciding the type of each of the parameters previously). But now it's all okay, noted in the future to create all new files in typescript instead of JS

Reply:
Again, you need to prioritise BOTH functionality + SWE principles when coding at the same time as a software engineer. This will save your time as well and not miss such a thing when you submit for review.